pkg/snapstore/gcs_snapstore.go

```diff
@@ -283,14 +283,15 @@ func (s *GCSSnapStore) componentUploader(wg *sync.WaitGroup, stopCh <-chan struc
 	}
 }
 
-// List will return sorted list with all snapshot files on store.
-func (s *GCSSnapStore) List() (brtypes.SnapList, error) {
+// List will return a sorted list with all snapshot files on store, excluding those marked with x-ignore-etcd-snapshot-exclude. Tagged snapshots can be included in the List call by passing true as the argument.
```

Comment:
```suggestion
// List will return a sorted list with all snapshot files on store, excluding those snapshots which metadata has tag: "x-ignore-etcd-snapshot-exclude". Tagged snapshots can be included in the List call by passing true as the argument.
```

Reply:
Addressed in [6b5b125](https://github.com/gardener/etcd-backup-restore/pull/776/commits/6b5b125f5bbb9044539f29992e62e52ac7a0bfa1).